### ISSUE TITLE
Missing aircraft sound folder

### ISSUE THREAD

**Issue by youngblood12345** [GaijinEntertainment/fmod_studio_warthunder_for_modders#305]
Im trying to find the kill notificaiton sound for Air battles and i noticed the folder its supposed to be in is completly missing
![image](https://github.com/user-attachments/assets/2b76f4dd-4f01-4c7b-bfbf-9b052fdd521d)

**Comment by RINZLER1999** [comment]
Yeah I have the same problem, I am trying to find the music files which I understand to be in the aircraft folder.

**Comment by RINZLER1999** [comment]
oh silly me, the folder shows up when you import the whole sound project that comes with the whole repository. Though it isn't obvious this is what you need to do.

**Comment by Uchuujin01** [comment]
> Im trying to find the kill notificaiton sound for Air battles and i noticed the folder its supposed to be in is completly missing ![image](https://private-user-images.githubusercontent.com/95578931/357532248-2b76f4dd-4f01-4c7b-bfbf-9b052fdd521d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0NDkxMjYsIm5iZiI6MTcyNzQ0ODgyNiwicGF0aCI6Ii85NTU3ODkzMS8zNTc1MzIyNDgtMmI3NmY0ZGQtNGYwMS00YzdiLWJmYmYtOWIwNTJmZGQ1MjFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI3VDE0NTM0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0N2MxMTQxMDBmMWMyN2VhNzk2OGE2MmIxNzNkODg0ZWZkMTRiZmMwYmNmN2QzYjhiMTIxYzE0MjNiMTdiMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UUPwThCXk-QT-fZgmw3qMjyVgzomIjFcbfCTbnkYwcU)

All sounds in project, make sure you cloned the repository. As far as I know it is not always can be zipped and download correctly.
